chat-sdk-ui/src/main/java/co/chatsdk/ui/threads/ThreadsFragment.java

```diff
@@ -152,11 +156,37 @@ public void setTabVisibility (boolean isVisible) {
         reloadData();
     }
 
+    List<Thread> replaceThread(List<Thread> threads, Thread thread) {
```

Comment:
I'm not clear about what this method does - could you add a comment to explain what it does and where it could be used?